canonical/ubuntu.com
# Create ubuntu.com/20years

Don't merge until Monday

## Done

- Build /20years based on [design](https://www.figma.com/design/8PJYpkyH1RP1RNDos6hzou/Ubuntu.com%2F20-years?node-id=0-1&t=1wmIt65KnblZy2VX-0) and [copydoc](https://docs.google.com/document/d/1vlENASSxNJdnK5tRuNYEZGVbh_qrVN0lhon8SOYo6A0/edit)
- Create `p-tube-line` class


## QA

- Open [the demo](https://ubuntu-com-14236.demos.haus/20years)
- Compare against [design](https://www.figma.com/design/8PJYpkyH1RP1RNDos6hzou/Ubuntu.com%2F20-years?node-id=0-1&t=1wmIt65KnblZy2VX-0) and [copydoc](https://docs.google.com/document/d/1vlENASSxNJdnK5tRuNYEZGVbh_qrVN0lhon8SOYo6A0/edit)
- Check on different screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14124
